docs/en/integrations/data-ingestion/kafka/kafka-clickhouse-connect-sink.md

```diff
@@ -25,8 +25,8 @@ The [Kafka Connect](https://docs.confluent.io/platform/current/connect/index.htm
 ### Version compatibility matrix
 
 | ClickHouse Kafka Connect version | ClickHouse version | Kafka Connect | Confluent platform |
-| -------------------------------- | ------------------ | ------------- | ------------------ |
-| 1.0.0                            | > 22.5             | > 2.7         | > 6.1              |
+|----------------------------------|--------------------|---------------|--------------------|
+| 1.0.0                            | > 23.8             | > 2.7         | > 6.1              |
```

Comment:
We must support 23.3 since it's a LTS version https://github.com/ClickHouse/ClickHouse/blob/master/SECURITY.md
```suggestion
| 1.0.13                            | >= 23.3             | >= 2.7         | >=> 6.1              |
```